docs/PARALLEL_COMPILATION.md

```
@@ -0,0 +1,370 @@
+    Path            string           // Import path
+    Dir             string           // Directory path
+    Kind            uint8            // ModuleKind (source/builtin/stdlib/external)
+    NoStd           bool             // Module doesn't import stdlib
+    HasModulePragma bool             // Has explicit module pragma
+    ImportPaths     []string         // Imported module paths
+    FilePaths       []string         // Source file paths
+    FileHashes      []project.Digest // Individual file content hashes
+    ContentHash     project.Digest   // Combined content hash
+    ModuleHash      project.Digest   // Module + dependencies hash
+    DependencyHash  project.Digest   // Hash of all dependency hashes
+    Broken          bool             // Module has errors
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)